sample-tracking-status-overview-domain/src/main/groovy/life/qbic/business/project/subscribe/SubscribeProject.groovy

```diff
@@ -28,6 +28,9 @@ class SubscribeProject implements SubscribeProjectInput {
         try {
             InputValidator.validate(subscriber, projectCode)
         } catch (ValidationException validationException) {
+            //todo this should never happen in production
+            //better: show notification to the user --> subscriptionFailed telling him to contact zendesk bzw this needs
+            //to be caught in the view otherwise
```

Comment:
Nice idea, please implement the notification right away 😄 this way it is not forgotten about.

Reply:
Is it better to catch it in the use case or in the view?

Reply:
On another thought maybe the exception is more fitting here. This is not a path we want to support so an exception is reasonable

Reply:
But still the app needs to account for that error and not let the software fail. The user might want to contact the support team since his desired project was not able to be subscribed to. Which should never happen, because then we show faulty projects

Reply:
What happens currently?

Reply:
An exception is thrown in the terminal and in the application the purble ! is shown.